sunpy/util/scraper.py

```diff
@@ -74,9 +74,9 @@ def __init__(self, pattern, **kwargs):
                 end=self.pattern[milliseconds.end():]
             ))
 
-        self.appendLogin = True
-        if 'appendLogin' in kwargs:
-            self.appendLogin = kwargs['appendLogin']
+        self.append_login = True
+        if 'append_login' in kwargs:
```

Comment:
why not add this as an explict kwarg to the signature?

Reply:
Right now, there was this one isolated case where we needed the scraper to not append the login details to the FTP URL. I thought it'll be better if we added this as a kwarg rather than make changes to the function prototype.

Reply:
tbh I am a little confused by why we are doing this at all. Why can't we append the login in the things that call the scraper that need it?

Reply:
Right now, every FTP URL that called the scraper was being appended with the login. If  you think that it'll be better to let the URLs decide if they need the login appended, then I believe that passing append_login as an explicit kwarg would be better. May I make it so?

Reply:
I think if the URLs need the login appended it should be appended by the caller before it gets to scraper, I don't understand why this code is in scraper at all lol. @dpshelio ?